### PR TITLE
Feature/label position

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: themes/theme-core/src/components/Field.style.ts [hunk 1]
```diff
@@ -2,9 +2,11 @@ import type { Theme } from '@marigold/components';
 
 export const Field: Theme['components']['Field'] = {
   base: {
-    display: 'flex',
-    flexDirection: 'row',
-    gap: 'xsmall',
-    alignItems: 'baseline',
+    // display: 'flex',
```
[the remark this answers is not in the document]
Comment (OsamaAbdellateef) [reply themes/theme-core/src/components/Field.style.ts]: Solved

**Comment by OsamaAbdellateef** [comment]
> @OsamaAbdellateef @sarahgm We have to discuss if we want to do it this way. If I got it correctly it is no longer possible to set a default per theme only via prop, correct?

What do you mean by setting default per theme !

**Comment by sebald** [comment]
> > @OsamaAbdellateef @sarahgm We have to discuss if we want to do it this way. If I got it correctly it is no longer possible to set a default per theme only via prop, correct?
> 
> What do you mean by setting default per theme ! 
> 

We need a way to configure a default placement per theme. In the core the label is always on the left, so whenever a form is created they would need to set it manually. It would be great if we can do this inside the theme.

For example `theme.defaults.labelPositon` or as part of the `FieldBase.style`. not sure what would be better.

**Comment by sebald** [comment]
Closing this since we don't need this anymore when using tailwind 🙈

**Pull request closed without merging**